roofit/roofitcore/src/RooDataHist.cxx

```diff
@@ -1664,9 +1682,9 @@ const std::vector<double>& RooDataHist::calculatePartialBinVolume(const RooArgSe
 {
   // Allocate cache if not yet existing
   vector<Double_t> *pbinv = _pbinvCacheMgr.getObj(&dimSet) ;
```

Comment:
Okay, I will add the same comment as in the git log!

Unfortunately I can't comment out only
```C++
vector<Double_t> *pbinv = _pbinvCacheMgr.getObj(&dimSet) ;
```
because the function returns a reference to a vector in the cache. I need to change a few other lines to make it work, but I'll do that.